lib/pages/saved_places/saved_places.dart

```diff
@@ -0,0 +1,155 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:latlong/latlong.dart';
+
+import 'package:trufi_core/blocs/search_locations/search_locations_cubit.dart';
+import 'package:trufi_core/l10n/trufi_localization.dart';
+import 'package:trufi_core/pages/saved_places/location_tiler.dart';
+import 'package:trufi_core/widgets/dialog_edit_text.dart';
+
+import '../../trufi_models.dart';
+import '../../widgets/trufi_drawer.dart';
+import '../choose_location.dart';
+
+class SavedPlacesPage extends StatelessWidget {
+  static const String route = '/places';
+
+  const SavedPlacesPage({Key key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final searchLocationsCubit = context.read<SearchLocationsCubit>();
+    final localization = TrufiLocalization.of(context);
+    final theme = Theme.of(context);
+    return Scaffold(
+      appBar: AppBar(title: Text(localization.menuYourPlaces)),
+      body: DefaultTabController(
+        length: 2,
+        child: Column(
+          children: [
+            const SizedBox(height: 10),
```

Comment:
I would not recommend this, if you press and hold the Tab the Material Ripple effect looks odd. Instead, increase the padding of the children on top by ten.
<img width=200 src="https://user-images.githubusercontent.com/8026644/117561352-75086700-b096-11eb-95f6-299dd339f85d.png">